### PR TITLE
Updated styles for NL page.

### PR DESCRIPTION
**Pull request by dwnoble**
* Updated page name from `NL Explorer` to `Chat`
* Added welcome message
* Added sidebar scaffolding. I'll replace the hardcoded queries with actual query history in a future PR
* Style updates

Screenshot
Screenshot 2023-07-17 at 10 40 04 PM

### CODE REVIEW

**Review by chejennifer — APPROVED** [review]
this looks nice! just some small suggestions

**Review by pradh — COMMENTED** [review]
Dan, this is nice!

Could you confirm that the various demos work ok with this change?

go/dcnl-main-demo
go/dcnl-sdg-demo
go/dcnl-ct-demo
go/dcnl-me-demo

For these please change the URL to localhost:8080/.

In the last one, s/Which cities in the San Francisco Peninsula have the most parks
 /Which cities in the SF Bay Area have the highest larceny/

**Comment by dwnoble** [comment]
> Dan, this is nice!
> 
> Could you confirm that the various demos work ok with this change?
> 
> go/dcnl-main-demo go/dcnl-sdg-demo go/dcnl-ct-demo go/dcnl-me-demo
> 
> For these please change the URL to localhost:8080/.
> 
> In the last one, s/Which cities in the San Francisco Peninsula have the most parks /Which cities in the SF Bay Area have the highest larceny/

Thanks! these all are working for me on localhost

**Comment by pradh** [comment]
> > Dan, this is nice!
> > Could you confirm that the various demos work ok with this change?
> > go/dcnl-main-demo go/dcnl-sdg-demo go/dcnl-ct-demo go/dcnl-me-demo
> > For these please change the URL to localhost:8080/.
> > In the last one, s/Which cities in the San Francisco Peninsula have the most parks /Which cities in the SF Bay Area have the highest larceny/
> 
> Thanks! these all are working for me on localhost

Great, thanks for checking!

**Review by pradh — APPROVED** [review]

File: static/js/apps/nl_interface/app.tsx [hunk 1]
```diff
@@ -203,24 +205,33 @@ export function App(): JSX.Element {
...
-          placeDetector={placeDetector}
-          setIndexType={setIndexType}
-          setDetector={setDetector}
-          setPlaceDetector={setPlaceDetector}
-        />
-        {isStartState && <QueryHistory onItemClick={onQueryItemClick} />}
+      <Sidebar />
+      <div className="chat-container">
+        <div className="chat-body" id="results-thread-container">
+          <QueryWelcome />
```
[the remark this answers is not in the document]
Comment (chejennifer) [reply static/js/apps/nl_interface/app.tsx]: that makes sense to me!

**Pull request merged**